translations/messages.de.yaml

```diff
@@ -9,6 +9,8 @@ ngsite.layout.cookie_settings.accepted: "Akzeptiert"
 ngsite.layout.cookie_settings.not_accepted: "Nicht akzeptiert"
 ngsite.layout.sponsored: 'Gesponsert'
 
+ngsite.general.read_more_about: 'Mehr erfahren über'
```

Comment:
Please rename this to `ngsite.read_more_about`, since we're using that format in other PRs.

Reply:
https://github.com/netgen/media-site/pull/219/commits/9fe1d519565ee64b53de0a0ea5ed3154b9c8bf07

renamed label for translation